### DISCUSSION TITLE
[client] Queries initially slow after computer goes to sleep, internet disconnect/reconnect

### DISCUSSION THREAD

**Discussion by huumn** [apollographql/apollo-server#7708] [Q&A]
(This is presumably a client-side question, but apollo-client doesn't have discussions enabled.)

When I leave my site open in a tab, let my laptop go to sleep, then awaken my laptop and attempt to perform queries in the open tab, it's extraordinarily slow. I suspect it has something to do with the tab getting disconnected from the internet and perhaps my polling queries causing the client to go into some kind of failed state.

I couldn't find anything after searching so I figured I'd give this googlable title. Has anyone else dealt with this?

**Comment by trevor-scheer** [comment]
I'll raise this to the client team, but you're probably better off posting this in the community or the discord for visibility purposes.

https://community.apollographql.com
https://discord.gg/graphos
